docs/users/faq.md

```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
 are available. In addition, plugins that support BungeeCord but only require
 that they are installed on the server (nothing on the proxy) typically use
 the BungeeCord plugin messaging channel, which can be supported by BungeeQuack
-for Velocity 1.0.x or natively by Velocity 1.1.0.
\ No newline at end of file
+for Velocity 1.0.x or natively by Velocity 1.1.0.
+
+# Common Errors
+
+``
+Can't connect to server lobby: If you wish to use IP forwarding, 
+please enable it in your Bungeecord config as well! 
+``
+
+``
+Can't conenct to server lobby: Your server did not send a forwarding request to the proxy. 
+Is it set up correctly?
+``
+
+These errors are result of improper configuration. See 
+[Player Information Forwarding](/wiki/users/forwarding/) 
+for a proper IP forwarding guide.
+
+``
+Can't connect to server lobby: This server requires you to connect with Velocity
+``
+
+This error is a result of only enabling forwarding on the backend server. To fix this error, 
+ensure that you have set up the correct player information forwarding method on the proxy.
```

Comment:
> correct player information forwarding method
This sounds a bit wordy.

Reply:
Perhaps "correct forwarding method"?

Reply:
I think "ensure you have set up player info(rmation) forwarding on the proxy" would be sufficient and concise.

Reply:
well, at that rate we should just leave it how it is... no?
It's not like we have a ton going on for this part of the documentation so we can get away with a little more precision.